app/ingest_main/Dockerfile-ingest_main

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+MAINTAINER James Lamb <jaylamb20@gmail.com>
+
+# TODO (jaylamb20@gmail.com):
+# Implement main ingestion service that reads from S3 and writes to our DB
```

Comment:
again, logstash s3 [input](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-s3.html) and es [output](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html) are really the simplest ways to ingest into ES. I can talk more tomorrow when we meet - I can quickly set this ingestion pipeline through that - we dont need a service IMO for this.